e2e-tests/monitoring-2-0/compare/statefulset_monitoring-pxc-oc.yml

```diff
@@ -67,7 +104,19 @@ spec:
               value: localhost
             - name: DB_PORT
               value: "3306"
+            - name: PMM_AGENT_PRERUN_SCRIPT
+              value: sleep 10; pmm-admin add $(DB_TYPE) --skip-connection-check --metrics-mode=push --username=$(DB_USER) --password=$(DB_PASSWORD) --cluster=$(DB_CLUSTER)  --service-name=$(PMM_AGENT_SETUP_NODE_NAME) --host=$(DB_HOST) --port=$(DB_PORT) $(DB_ARGS); pmm-admin annotate restart
```

Comment:
That code no longer generates `sleep 10`. Not sure how those tests work

Reply:
fixed